docs/monitors/_monitor-config.md

```diff
@@ -0,0 +1,174 @@
+
+<!--- Generated by to-integrations-repo script in Smart Agent repo, DO NOT MODIFY HERE --->
+<!--- GENERATED BY gomplate from scripts/docs/templates/monitor-main.md.tmpl --->
+
+# Monitor Configuration
+
+Monitors gather metrics from the host and from running applications.  They are
+configured in a list called `monitors` in the [main agent config
+file](../config-schema.md).  For example:
+
+```yaml
+monitors:
+ - type: elasticsearch
+   discoveryRule: 'port == 9200 && container_image =~ "elasticsearch"'
+ - type: cpu
+ - type: docker-container-stats
+   dockerURL: tcp://localhost:2379
+```
+
+[See here for a list of available monitors](#monitor-list)
```

Comment:
"See <link to target> for a list of available monitors."

Reply:
I didn't edit this file. I won't change it.